tools/reviewer/webapp/src/app/dashboard/file-changes/commit-menu/commit-menu.component.ts

```diff
@@ -0,0 +1,211 @@
+import { Component, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+
+import { MouseService } from '@/core/services';
+import { LoadService, StateService } from '../services';
+import { CommitInfo } from './commit-popup';
+
+// Drag element is an orange circle (or square), which can be drag and droped by user
+// to pick another commit.
+// Point is an white circle (or square), where can be dropped drag element.
```

Comment:
The comment seemed a bit funky here. I would suggest using `which can be dragged and dropped by user` and similarly in the next comment `where a dragged element (dragable?) can be dropped`